src/Http/Client/Response.php

```diff
@@ -119,6 +120,20 @@ class Response extends Message implements ResponseInterface
      */
     protected $_json;
 
+    /**
+     * Stream mode options.
+     *
+     * @var string
+     */
+    protected $_streamMode = 'wb+';
+
+    /**
+     * Stream target or resource object.
+     *
+     * @var string|resource
+     */
+    protected $_streamTarget = 'php://memory';
```

Comment:
Why is the response needing to know about the stream properties?

Reply:
> What is the use case around serializing a server response? I can see the use case for a client response but not a server response.

In fact I find it more useful for a `Cake\Http\Client\Response` instance (this is just what I would need), rather than for a `Cake\Http\Response` instance.

I said to make it possible even for a server response just because the code would be the same. I added the `$_streamTarget` property just for this reason: to make the two classes homogeneous, not because we really need them. Actually this is not necessary.